packages/mc-dev-authentication/middlewares/create-mc-dev-authentication-middleware.js

```diff
@@ -0,0 +1,84 @@
+const path = require('path');
+const pug = require('pug');
+const { logout } = require('../routes');
+
+const compileLoginView = pug.compileFile(
+  path.join(__dirname, '../views/login.pug')
+);
+const compileLogoutView = pug.compileFile(
+  path.join(__dirname, '../views/logout.pug')
+);
+
+function createMcDevAuthenticationMiddleware(applicationConfig) {
+  const htmlLogin = compileLoginView({ env: applicationConfig.env });
+  const htmlLogout = compileLogoutView({ env: applicationConfig.env });
```

Comment:
Instead of relying of writing/reading these pages from disk, we can simply handle it in memory.

Here we compile the pages and serve them when needed.